File: codekata/basics/scalene_traingle_198.py
def checkvalid(a,b,c):
    if((a!=b) and (b!=c) and (a!=c)):
        return True
    else:
        return False
a,b,c=map(int,input().split())
if checkvalid(a,b,c):
    print("yes")
else:
    print("no")

File: javascript/basics/scalene_traingle_198.js
const readline=require('readline');
const inp=readline.createInterface({input:process.stdin});
inp.on("line",(data)=>{
    var n=data.split(' ');
    var a=n[0];
    var b=n[1];
    var c=n[2];
    if((a!=b) && (a!=c) && (b!=c)){
        console.log("yes");
    }
    else{
        console.log("no");
    }
});
